Makefile

```
@@ -315,7 +318,20 @@ lint-deps: ## Validate architecture/import boundaries with dependency-cruiser
+
+lint-prod-guardrails: ## Enforce the production-safety invariants (privileged-workflow alerting, fail-closed edge routing, no source maps)
+	$(PM_EXEC) node scripts/ci/lint-prod-guardrails.mjs
+
+# lint-security-txt and lint-prod-guardrails DO belong in the aggregate below,
+# unlike lint-contracts and lint-metrics: both read only committed files (no
+# network, no host binary, no Docker), so they are hermetic and cannot make the
+# static lane flaky. lint-prod-guardrails additionally joins CI_LINT_TARGETS
+# because it needs `node` + js-yaml, which the parallel ci-lint runner provides;
+# lint-security-txt is pure bash and needs no package manager, mirroring how
+# lint-deps stays out of that list.
+lint: lint-next lint-tsc lint-md lint-deps lint-security-txt lint-prod-guardrails ## Runs all linters: ESLint, TypeScript, Markdown, dependency-cruiser, the RFC 9116 security.txt gate, and the production-safety guardrails in sequence.
```

checkmake (0.3.2)

[warning] Target "lint" should be declared PHONY.

(phonydeclared)